lib/public_include/fty_shm.h

```diff
@@ -107,12 +130,22 @@ class shmMetrics
     std::vector<fty_proto_t*> m_metricsVector;
 };
 
-// C++ versions of fty_shm_write_metric()
+// C++ versions of fty_shm_write_metric_proto()
+// @param metric : metric data will be written to file
```

Comment:
at this level, the metric file model is completely private... the user don't know 'metric file'. It's not relevant here
same for @return descr.